modules/sdl/sdl.c

```diff
@@ -140,6 +140,7 @@ static void destructor(void *arg)
 
 	/* needed to close the window */
 	SDL_PumpEvents();
+	SDL_VideoQuit();
```

Comment:
is it correct to quit video in the destructor ?

does it work to create windows 2,3,4 etc times ?

Reply:
My fault, will fix this this.